src/main/java/org/openstreetmap/atlas/utilities/testing/TestAtlasHandler.java

```diff
@@ -91,25 +103,33 @@ public static Atlas getAtlasFromJsomOsmResource(final boolean josmFormat,
      *
      * @param pbfResource
      *            The pbf input resource to use
+     * @param iso
+     *            ISO code to be applied to all features
      * @return the resulting Atlas
      */
-    private static Atlas buildAtlasFromPbf(final Resource pbfResource)
+    private static Atlas buildAtlasFromPbf(final Resource pbfResource, final String iso)
     {
         // Create raw Atlas
         final AtlasLoadingOption loadingOption = AtlasLoadingOption.withNoFilter();
+        if (!iso.equals(ISOCountryTag.UNKNOWN_ISO_COUNTRY))
+        {
+            loadingOption.setCountrySlicing(true);
+            loadingOption.setCountryBoundaryMap(CountryBoundaryMap
+                    .fromBoundaryMap(Collections.singletonMap(iso, MultiPolygon.MAXIMUM)));
+        }
         final Atlas rawAtlas = new RawAtlasGenerator(pbfResource, loadingOption,
                 MultiPolygon.MAXIMUM).build();
 
-        // Way-section
-        return new WaySectionProcessor(rawAtlas, loadingOption).run();
+        // Country Slice and Way-Section
+        return new WaySectionProcessor(iso.equals(ISOCountryTag.UNKNOWN_ISO_COUNTRY) ? rawAtlas
+                : new RawAtlasCountrySlicer(loadingOption).slice(rawAtlas), loadingOption).run();
```

Comment:
Is the fact that we're now country-slicing something worth calling out? Feels like that might have some downstream impacts on JOSM-atlas tests

Reply:
I don't think it should really have any impact because this is using rectangle.MAXIMUM as the one "country" in the country boundary map. So no actual slicing should occur. But I would agree it is worth calling out.